### DISCUSSION TITLE
[Application] Can't get my Github Education approved, because i didn't have started classes yet.

### DISCUSSION THREAD

**Discussion by szpolny** [community/community#132498] [GitHub Education]
### Hi everyone,

Hi everyone, I am in the middle of summer before my first semester in University. I have just finished High School in my local country, and for further studying I am moving to Netherlands. I study programming a lot for my own and I use a lot of benefits from Github Education. But my access to benefits have just expired and I can't get verified. In Poland, every school issue ID to the end of September, so my is still valid, but i can't get verified with it. I try to get verify with enrollment letter from my future school, but application is also rejected because i am not in the country where my school is located. Is there an option to get verified in my current position?

**Comment by davevad93** [comment]
Hi @szpolny ,

As stated in the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) , In order to qualify for the GitHub education program, you need to be either located close to your campus, or prove that you are a part of a verifiable distance learning program. So I believe you'll have to wait until you start your classes in the Netherlands and then reapply using the university documentation. Otherwise you'll be probably rejected for the following reason:

- ***You appear to not be near any campus location for the school you have selected.***

**Comment by SolutionsCrafter** [comment]
Hey! I'm Channa,

I know it can be really frustrating to deal with this kind of situation.Here’s what you can do:

First, reach out to GitHub support and explain what’s going on. Let them know you’re in the summer before starting university in the Netherlands and that your current student ID is valid until September, but you’re facing issues with verification. 
When you contact them, make sure to attach your current student ID and the enrollment letter from your future school. Explain that you’re not yet in the Netherlands but will be moving there soon for your studies.
Clearly outline that you’re in a transition period and need access to the GitHub Education benefits to continue your work. Mention how important these resources are for your studies and projects.
If you don’t get a response in a few days, follow up with another message to check on the status of your request.

Hope this will helps you and good luck with your studies!
